installing/installing_aws/installing-aws-user-infra.adoc

```diff
@@ -59,6 +59,8 @@ include::modules/installation-generate-aws-user-infra-install-config.adoc[levelo
 
 include::modules/installation-configure-proxy.adoc[leveloffset=+2]
 
+include::modules/installation-three-node-cluster.adoc[leveloffset=+2]
+
```

Comment:
Since the title of the story is `NE-247 Support 3-node bare metal clusters with the routers` (see also: https://issues.redhat.com/browse/NE-247) , and QE just ensured  this can be supported on **Bare Metal** , so I don't think we should add it to other platform like aws, azure etc.

Reply:
Sounds good! I'll comment out the includes for everything but bare metal. It will only build in bare metal, but the other includes will remain in place for when they're available.